tests/unit/test_reservation.py

```diff
@@ -542,7 +550,7 @@ def test___session_not_created___get_connection_with_session_type_object___conne
     assert connection.session is None
 
 
-def test___no_connections___get_connection_with_session_type_object___value_error_raised(
+def test___no_connection___get_connection_with_session_type_object___value_error_raised(
```

Comment:
"no connections" is fine. It doesn't have to match get_connection. It's not a big deal though.

https://englishlessonsbrighton.co.uk/followed-singular-plural-noun/